src/pages/ReportParticipantsPage.js

```diff
@@ -75,7 +75,7 @@ const getAllParticipants = (report, personalDetails) => {
             tooltipText: userLogin,
             participantsList: [{login, displayName: userPersonalDetail.displayName}],
         });
-    });
+    }).value();
```

Comment:
```suggestion
    }).sortBy(participant => participant.displayName.toLowerCase()).value();
```

I think we should sort participants by displayName, that's how we do it on [WorkspaceMembersPage](https://github.com/Expensify/App/blob/a7863493a03970cdc0687621ec40dc75aea99477/src/pages/workspace/WorkspaceMembersPage.js#L329)

And `.toLowerCase()` to avoid issues similar to https://github.com/Expensify/App/issues/16925

Reply:
There is also no sorting on [WorkspaceInvitePage](https://github.com/Expensify/App/blob/a7863493a03970cdc0687621ec40dc75aea99477/src/pages/workspace/WorkspaceInvitePage.js#L148), we can add it so it's consistent across the screens

Reply:
Sounds good @eVoloshchak, I'll add these changes.